style.css

```diff
@@ -415,6 +420,65 @@ hr.mobile {
   display: none;
 }
 
+.modal-mobile,
+.modal-desktop {
+  position: fixed;
+  top: 0;
+  background-color: #fff;
+  z-index: 999;
+  padding: 5px 12px 24px 12px;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
```

Comment:
- [ ] Please make sure to add top and left-right margins to the model for the mobile screens.

### Your design

![image](https://user-images.githubusercontent.com/83803414/209133155-a5f764a9-8a1b-460e-8363-b9e92d2a5ef6.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/209132795-cdaf1451-b4e4-4f0c-a4f9-7b31caa517ba.png)